### PR TITLE
Creating product landing page

### PR DESCRIPTION
**Pull request by MasomRezaie**
This a project is built by html and CSS.
I configured linters.
I tested in freecodecamp.org.

### CODE REVIEW

File: index.html
Comment (ahzamir) [inline index.html]: - Your section does not have a closing tag.

File: README.md
Comment (ahzamir) [inline README.md:1]: - Here you guided the user on how to create a product landing page, instead, you have to guide the user on what your project is about, how to use it, how you made it, and all the info about it.

File: styles.css
Comment (ahzamir) [inline styles.css:1]: - **[OPTIONAL]** - Although your code passes the freecodecamp tests, I suggest adding more content to your webpage and some design to make it look more awesome to share.

**Review by ahzamir — CHANGES_REQUESTED** [review]
### Hi @MasomRezaie 🙋‍♀️,

> Excellent work! You have done a great job implementing the project requirements but your project still misses some of the requirements:

- Although your code is passing all the checks on the freecodecamp website, but you need to work on your project in order to get the final approval.
  - Check the inline comments below for more details.
  
# Status: Required Changes 🔁
Make changes to your code and commit and request for a re-review.

AlmostThereReachGIF

 ## Highlights 🌞
 
✔️ Simple Design implemented.
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed and completed the Product Landing Page and all requirements.
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
❌Your code is clean
❌Your project has a professional README.md file

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Comment by MasomRezaie** [comment]
@ahzamir Hi sir, hope you are doing well. I applied your feedback. Please re-review.

**Review by ahzamir — APPROVED** [review]
### Hi @MasomRezaie 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text on it with a screenshot.

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

GreatJobThumbsUpGIF

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Product Landing Page project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your project has a professional README.md file

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Comment by ahzamir** [comment]
@MasomRezaie 

- Why you did not merge your branch after getting the approval?

**Pull request merged**
